contracts/interfaces/for-export/IAedexV2Factory.aes

```diff
@@ -44,8 +44,9 @@ contract interface IAedexV2Pair =
   stateful entrypoint transfer_allowance : (address, address, int) => unit
 
 main contract AedexV2Factory =
-  record state = {fee_to : option(address),fee_to_setter : address,pairs : map(IAEX9Minimal, map(IAEX9Minimal, IAedexV2Pair)),all_pairs : list(IAedexV2Pair),pair_model : IAedexV2Pair,allow_debug_mode : bool}
+  record state = {fee_to : option(address),fee_to_setter : address,pairs : map(AedexV2Factory.tokenIdPair, IAedexV2Pair),all_pairs : list(IAedexV2Pair),pair_model : IAedexV2Pair,allow_debug_mode : bool}
   datatype event = PairCreated(address, address, address, string)
+  type tokenIdPair = (IAEX9Minimal * IAEX9Minimal)
```

Comment:
This type is also defined in `contracts/AedexV2Factory.aes`. Is there way to define it only once instead of duplicating the definition?

Reply:
All `for-exports/*` are autogenerated , so not only this type is defined twice, all types will be re-defined because the scope of `for-exports` interfaces is to be used isolated in external projects, without `include` directives.